### ISSUE TITLE
document access for foreign users

### ISSUE THREAD

**Issue by loveshack** [N8-CIR-Bede/documentation#88]
I don't know whether this should go on the web site or here, but I've
been asked two or three times about access for foreign users.  Perhaps
that's in the info that PIs get sent, but it's relevant before a
project application.  When I asked before, the only requirement seemed
to be having a SAFE account, but I don't know what conditions there
might be on that.

**Comment by ptheywood** [comment]
Any clarifications to this would be worthwhile including on the main site, and probably duplicating to the relevant page(s) in this documentation if it is something that is repeatedly asked.

For the n8cir.org.uk site, the relevant page would probably be:

https://n8cir.org.uk/supporting-research/facilities/bede/docs/bede_registrations/

This already details that access is via N8 member institutes, consortia such as HECBioSim or EPSRC calls (which implies UK-led projects?). 

For the queries you've had re: international users I assume this has been mostly as users for projects that otherwise would be approved under N8 institutes / EPSRC calls led by a UK institute? 

I don't know the actual answer for this, @MarionBWeinzierl or @bodgerer may have a concrete answer?

**Comment by MarionBWeinzierl** [comment]
Hm, I believe the answer is that projects need to fall into one the categories listed on the page that you linked to, but then the PI of that project is free to approve non-UK users.

In a different context I have seen that there might be some configuration issues for non-UK users in that SAFE workspace which, in that case, needed to be sorted. I'd just try registering the user, and if anything comes up sort it out then.

**Comment by loveshack** [comment]
Yes, I didn't know where it should go -- one of the problems of having
information in multiple places, but it might be considered an FAQ.  The
queries were about projects with a Manchester PI and arbitrary
non-Manchester collaborators.  (I said just to try creating in account
in the absence of any indication it might be rejected.)
